sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -117,26 +118,29 @@ def plot(self, include_values=True, cmap='viridis',
         return self
 
 
-def plot_confusion_matrix(estimator, X, y_true, labels=None,
+def plot_confusion_matrix(estimator=None, X=None, y_true=None, labels=None,
                           sample_weight=None, normalize=None,
                           display_labels=None, include_values=True,
                           xticks_rotation='horizontal',
                           values_format=None,
-                          cmap='viridis', ax=None):
+                          cmap='viridis', ax=None, y_pred=None):
     """Plot Confusion Matrix.
 
     Read more in the :ref:`User Guide <confusion_matrix>`.
 
     Parameters
     ----------
-    estimator : estimator instance
-        Trained classifier.
+    estimator : estimator instance, default=None
+        Trained classifier. Either `estimator` and `X` must be specified, or
+        `y_pred`.
 
-    X : {array-like, sparse matrix} of shape (n_samples, n_features)
-        Input values.
+    X : {array-like, sparse matrix} of shape (n_samples, n_features),
+        default=None
```

Comment:
You need to add a `\` at the end of the previous line. Also please indent this one further right